client/buildpacks.go

```diff
@@ -125,16 +126,20 @@ func (b *buildpacksFunctionServer) run() (func(), error) {
 	if err != nil {
 		return nil, err
 	}
-
+	var temp = b.envs.String()
+	log.Printf("Extra variable s%se",temp)
 	args := []string{"docker", "run",
 		"--network=host",
 		// TODO: figure out why these aren't getting set in the buildpack.
 		"--env=FUNCTION_TARGET=" + b.target,
 		"--env=FUNCTION_SIGNATURE_TYPE=" + b.funcType,
-		image,
 	}
+	if len(*b.envs) != 0 {
+		args = append(args, *b.envs...)
+	}
+	args = append(args, image)
```

Comment:
[nit] does it work if we make image the first parameter after "docker run" so that we don't need to do this extra append?

Reply:
This does not work. exec checks for an executable with the name as that of first parameter.